docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-jumpcloud.md

```diff
@@ -0,0 +1,197 @@
+
+### Create an application in JumpCloud
+
+To use JumpCloud for your IdP, do the following:
+
+1. Sign in to JumpCloud using an administrator account.
+
+1. Create an application.
+
+    - Under **SSO Applications**, click **Add New Application**.
+    - Select **Custom Application**, and make sure the integration supports "SSO with OIDC" on the next page.
+    - Under **Manage Single Sign-On (SSO)**, select **Configure SSO with OIDC**, and click **Next**.
+    - Under **Enter General Info**, add the application name (for **Display Label**), **Description**, and logo (for **User Portal Image**), and select **Show this application in User Portal**.
+    - Click **Configure Application**.
+
+1. Configure your application.
+
+    Under **SSO > Endpoint Configuration**, configure the following:
+
+    - **Redirect URIs** - enter `https://<your-YugabyteDB-Anywhere-IP-address>/api/v1/callback?client_name=OidcClient`.
+    - **Client Authentication Type** - select **Client Secret Post**.
+    - **Login URL** - enter `https://<your-YugabyteDB-Anywhere-IP-address>/login`.
+
+    Under **Attribute Mapping**, for **Standard Scopes**, select **Email** and **Profile**.
+
+    Click **Activate** when you are done.
+
+    You will be prompted in a pop up to save the **Client ID** and **Client Secret**. Save these in a secure location, you will need to provide these credentials in YugabyteDB Anywhere.
+
+1. Configure Attributes and Identity Management as required.
+
+1. Integrate the user in JumpCloud.
+
+    - Navigate to **User Groups**, select the user groups you want to access YugabyteDB Anywhere, and click **Save** when you are done.
+
+To [configure](#configure-yugabytedb-anywhere) JumpCloud federated authentication in YugabyteDB Anywhere, you need the following application properties:
+
+- **Client ID** and **Client Secret** of the application you created. These are the credentials you saved when you activated your application. The **Client ID** is also displayed on the **SSO** tab.
+
+For more information, refer to the [JumpCloud](https://jumpcloud.com/support/sso-with-oidc) documentation.
+
+### Configure YugabyteDB Anywhere
+
+To allow users to access their JWT from the YugabyteDB sign in page, you must enable the OIDC feature via a configuration flag before you configure OIDC.
+
+#### Enable OIDC enhancements
+
+To enable some features of the OIDC functionality in Yugabyte Anywhere, you need to set the `yb.security.oidc_feature_enhancements` configuration flag to true as follows:
+
+1. Navigate to **Admin > Advanced > Global Configuration**.
+
+1. Search on OIDC to display the configuration setting and set it to true.
+
+    ![Configuring yb.security.oidc_feature_enhancements flag for OIDC](/images/yp/security/oidc-azure-globalfeature.png)
+
+#### Enable OIDC authentication
+
+To configure YugabyteDB Anywhere for OIDC, you need to be signed in as a Super Admin. You need your JumpCloud application client ID and client secret.
```

Comment:
Would move to Configure YugabyteDB Anywhere cause you need superuser to set the global flag also I believe

Reply:
aah yes .. I made the change to azure as well.